### ISSUE TITLE
v0.3.0: "Cannot read property 'transform' of undefined while parsing file"

### ISSUE THREAD

**Issue by girvo** [garthenweb/bubleify#2]
```
$ npm run build

> skate-test@1.0.0 build /Users/josh/Work/Prototypes/react-buble-docker/code
> browserify -d -g uglifyify client/app.js | exorcist build/app.js.map > build/app.js

TypeError: Cannot read property 'transform' of undefined while parsing file: /Users/josh/Work/Prototypes/react-buble-docker/code/client/app.js
    at Bubleify._flush (/Users/josh/Work/Prototypes/react-buble-docker/code/node_modules/bubleify/lib/Bubleify.js:53:37)
    at Bubleify.<anonymous> (_stream_transform.js:118:12)
    at Bubleify.g (events.js:273:16)
    at emitNone (events.js:80:13)
    at Bubleify.emit (events.js:179:7)
    at prefinish (_stream_writable.js:468:12)
    at finishMaybe (_stream_writable.js:476:7)
    at endWritable (_stream_writable.js:488:3)
    at Bubleify.Writable.end (_stream_writable.js:453:5)
    at DuplexWrapper.<anonymous> (/Users/josh/Work/Prototypes/react-buble-docker/code/node_modules/duplexer2/index.js:29:14)
```

That's generated with the simplest code possible in the `client/app.js` file: 

```
/**
 * App entry-point
 */
'use strict'

function hello() {
  return "Hello!"
}
```

Removing `uglifyify` didn't fix it either, nor exorcist...

My `package.json` is as follows:

``` json
{
  "name": "react-buble",
  "version": "1.0.0",
  "description": "",
  "main": "src/index.js",
  "scripts": {
    "build": "browserify -d -g uglifyify client/app.js | exorcist build/app.js.map > build/app.js",
    "watch": "watchify -d -g uglifyify -o 'exorcist build/app.js.map > build/app.js' client/app.js",
    "dev": "nodemon -e js,html --ignore client/ src/index.js",
    "start": "node src/index.js",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "Josh Girvin <josh@jgirvin.com>",
  "license": "UNLICENSED",
  "dependencies": {
    "koa": "^1.2.0",
    "koa-mount": "^1.3.0",
    "koa-static": "^2.0.0",
    "react": "^15.1.0",
    "react-dom": "^15.1.0"
  },
  "devDependencies": {
    "babel-eslint": "^6.0.4",
    "browserify": "^13.0.1",
    "bubleify": "^0.3.0",
    "eslint": "^2.11.1",
    "eslint-config-standard": "^5.3.1",
    "eslint-plugin-promise": "^1.3.2",
    "eslint-plugin-react": "^5.1.1",
    "eslint-plugin-standard": "^1.3.2",
    "exorcist": "^0.4.0",
    "nodemon": "^1.9.2",
    "uglifyify": "^3.0.1",
    "watchify": "^3.7.0"
  },
  "browserify": {
    "transform": [
      "bubleify",
      "uglifyify"
    ]
  }
}
```

**Comment by garthenweb** [comment]
Thx for your report!

This seams to be somehow related to a change of the default export of bublè in version 0.10.6 that is not compatible with the babel transpiler anymore. I checked bublé versions smaller than 0.10.6 and they are still working.

Issue is fixed with bubleify v0.3.1.

**Comment by girvo** [comment]
Champion, thank you!
